department-of-veterans-affairs/caseflow
# Justin/appeals 13256

Resolves [APPEALS-13256](https://vajira.max.gov/browse/APPEALS-13256)

### Description
Please explain the changes you made here.

### Acceptance Criteria
- [ ] An “Add issue” button appears below the Issue section
- [ ] When selected, user can add a new issue 
- [ ] When adding an issue, user is required to select the Disposition by court from the following values:
- Abandoned
- Affirmed
- Dismissed
- Settled
- Reversed
- Vacated and Remanded
- N/A

- [ ] Added issues appear in an “Added issues” section on the dashboard
- This includes those existing on page load and those added while editing the CAVC Dashboard
- [ ] Added issues can also be removed

### Testing Plan
1. Log in as an OCC User, go to the cavc dashboard for a cavc remand, and verify the appeal can be seen. No "Add Issue" button should display for this user, as the appeal should be Read Only.
2. Log in as an OAI user, go to the cavc dashboard for a cavc remand, and verify the appeal can be seen. For this user, the button "Add Issue" should be visible on the bottom left of the dashboard. 
3. Click on the Add Issue button
4. Choose a benefit type and verify that the issue category is related to the benefit type. (The issue category list should be empty until a benefit type is selected). Then select a disposition by court option and submit. 
5. Verify that the issue is added under an "Added Issues" section on the dashboard. A "remove issue" button should be visible beneath the new added issue on the right side of the dashboard under the new issue.

### User Facing Changes
 BEFORE 
![image](https://user-images.githubusercontent.com/95875751/220787764-d8a6c172-2a90-432c-829e-c0126f1ac1d8.png)

---
AFTER
![image](https://user-images.githubusercontent.com/95875751/220787610-765b28ac-f141-4be8-b67b-8e7c6ddf99fd.png)
